packages/dhis2/src/Adaptor.js

```diff
@@ -273,16 +223,23 @@ export function create(resourceType, data, options = {}, callback = s => s) {
       );
     } else {
       promise = request(configuration, {
-        method: 'post',
-        url: generateUrl(configuration, resolvedOptions, resolvedResourceType),
-        params,
+        method: 'POST',
+        path: generateUrlPath(
+          configuration,
+          resolvedOptions,
+          resolvedResourceType
+        ),
+        options: resolvedOptions,
         data: resolvedData,
-        ...requestConfig,
       });
     }
 
     return promise.then(result => {
-      const details = `with response ${JSON.stringify(result.data, null, 2)}`;
+      const details = `with response ${JSON.stringify(
+        result?.body?.message,
+        null,
+        2
+      )}`;
       console.log(`Created ${resolvedResourceType} ${details}`);
```

Comment:
Do we really want to log this btw? Maybe we should just log the id?

Can you check the issues to see if anyone has complained about dhis2 logging too much data, and maybe ask Aleksa and Mtuchi about this?

My instinct is that we should **not**  be force-logging data

Reply:
It might technically be another issue to fix this, but because I've seen it now I want to ask the question here first